src/lib/Scenes/Map/Bucket.ts

```diff
@@ -1,15 +1,11 @@
 import { GlobalMap_viewer } from "__generated__/GlobalMap_viewer.graphql"
 import { filter } from "lodash"
 import moment from "moment"
-import { MapTab } from "./types"
 
-export type BucketKey = "saved" | "fairs" | "galleries" | "museums" | "closing" | "opening"
-export type BucketResults = { [key in BucketKey]: any[] }
-
-export const bucketCityResults = (viewer: GlobalMap_viewer): BucketResults => {
+export const bucketCityResults = (viewer: GlobalMap_viewer) => {
```

Comment:
OK, so I removed explicit typing all of the show here - all of the responses were correctly inferred in this function, so I could ditch the top one